modules/fieldlinessequence/rendering/renderablefieldlinessequence.h

```diff
@@ -53,17 +51,33 @@ class RenderableFieldlinesSequence : public Renderable {
     void render(const RenderData& data, RendererTasks& rendererTask) override;
     void update(const UpdateData& data) override;
 
+    static documentation::Documentation Documentation();
+
 private:
     // ------------------------------------- ENUMS -------------------------------------//
     // Used to determine if lines should be colored UNIFORMLY or by an extraQuantity
-    enum class ColorMethod : int {
+    enum class ColorMethod {
         Uniform = 0,
-        ByQuantity
+        ByQuantity = 1
+    };
+    enum class SourceFileType {
+        Cdf = 0,
+        Json = 1,
+        Osfls = 2
     };
 
     // ------------------------------------ STRINGS ------------------------------------//
-    std::string _identifier;                               // Name of the Node!
-
+    std::string _identifier;                               // Name of the Node! 
+    // cdf, osfls or json
+    SourceFileType _inputFileType;
+    // Output folder path in case of file conversion
+    std::string _outputFolderPath;
+    // which tracing vaiable to trace. 'b' for fieldline is default
+    std::string _tracingVariable;
+    // path to directory with seed point files
+    std::filesystem::path _seedPointDirectory;
+    // optional except when using json input
+    std::string _modelStr;
     // ------------------------------------- FLAGS -------------------------------------//
     // Used for 'runtime-states'. True when loading a new state from disk on another
     // thread.
```

Comment:
I think the `_mustLoadNewStateFromDisk` variable is necessary anymore.. it is only used in the `update` method and it doesn't seem to store any meaningful data across frames.  So by having it as a local variable in the `update` method, there is no need to keep it around for the remainder of the time

Reply:
Same argument for `_needsUpdate` i think